source/Nuke.GlobalTool/templates/_build.sdk.csproj

```diff
@@ -12,7 +12,7 @@
   <ItemGroup>
     <PackageReference Include="Nuke.Common" Version="_NUKE_VERSION_" />
     <PackageReference Include="Nuke.CodeGeneration" Version="_NUKE_VERSION_" />                  // GENERATION
-    <PackageReference Include="GitVersion.CommandLine.DotNetCore" Version="4.0.1-beta1-49" />    // GITVERSION
+    <PackageReference Include="_GIT_VERSION_PACKAGE_" Version="_GIT_PACKAGE_VERSION_" />         // GITVERSION
```

Comment:
I couldn't name it `_GIT_VERSION_PACKAGE_` and `_GIT_VERSION_PACKAGE_VERSION` because the templating seems to not look for the best (longest) match.

The version was just the package id again.